test/e2e/framework/framework.go

```diff
@@ -64,13 +71,16 @@ type Framework struct {
 	KubeClient clientset.Interface
 	ExtClient  apiextension.Interface
 	QdrClient  qdrclient.Interface
+	DynClient  dynamic.Interface
+	OcpClient  ocpClient
```

Comment:
I believe we should be up a level and use https://github.com/openshift/client-go/blob/2823239d2298214509c3536714f684101799e81b/route/clientset/versioned/clientset.go#L42
which will make the client work like the others. Also, suggest that maybe name shoud be RouteClient instead of OcpClient.

Reply:
Ok, I will try using the high-level one. The idea for RoutesClient, under OcpClient, was to use it as an aggregator for other OpenShift resource clients in the future. Does it make sense?